apps/notification/models.py

```diff
@@ -73,3 +75,31 @@ class Meta:
     @staticmethod
     def get_for(user):
         return Notification.objects.filter(receiver=user).distinct()
+
+
+class Assignment(models.Model):
+    """
+    Assignment Model
+    """
+    timestamp = models.DateTimeField(
+        default=timezone.now,
+    )
+    created_for = models.ForeignKey(User, on_delete=models.CASCADE)
+    project = models.ForeignKey(
+        Project,
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True,
+        default=None
+    )
```

Comment:
If others haven't pulled this change. Could you revert your migration in your system, remove old migrations files and create a fresh one where project null=True is not allowed?